function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautLambdaContainerHandler.java

```diff
@@ -49,10 +49,7 @@
 import io.micronaut.http.server.types.files.StreamedFile;
 import io.micronaut.inject.qualifiers.Qualifiers;
 import io.micronaut.jackson.codec.JsonMediaTypeCodec;
-import io.micronaut.web.router.MethodBasedRouteMatch;
-import io.micronaut.web.router.RouteMatch;
-import io.micronaut.web.router.Router;
-import io.micronaut.web.router.UriRouteMatch;
+import io.micronaut.web.router.*;
```

Comment:
Please, don't use wildcard imports. 

It is easy [disable wildcard imports in IntelliJ IDEA](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports).

Reply:
No, it is not a new rule. I just assumed we were doing that. Are you against avoiding wilcard imports @graemerocher ?

Reply:
A rule that is not in the check style is not a rule. There are wildcards used all over the codebase right now. 

If we all agree that this should be a rule then the check style needs updating to reflect that.

Reply:
Ok, I will bring this conversation to the group discussions. @pgressa do as you prefer regarding the wildcard imports.